client/templates/timeline/timeline.js

```diff
@@ -14,6 +14,10 @@ Template.timeline.helpers({
     },
     projects: function() {
         return Projects.find();
+    },
+    getProjectColor: function() {
```

Comment:
It's a left over from trying to get the span in the bottom to match the projects color. To be removed.